JuliaInterface/src/JuliaInterface.c

```diff
@@ -488,19 +524,6 @@ jl_function_t * get_function_from_obj_or_string(Obj func)
 }
 
 
-jl_module_t * get_module_from_string(char * name)
-{
-    // It suffices to use JULIAINTERFACE_EXCEPTION_HANDLER here, as
-    // jl_eval_string is part of the jlapi, so don't have to be wrapped in
-    // JL_TRY/JL_CATCH.
-    jl_value_t * module_value = jl_eval_string(name);
-    JULIAINTERFACE_EXCEPTION_HANDLER
-    if (!jl_is_module(module_value))
-        ErrorQuit("Not a module", 0, 0);
-    return (jl_module_t *)module_value;
-}
```

Comment:
Instead of moving this code around, you could also add a function declaration at the top of the file. Not that I care where it is, but moving functions complicates reviewing (e.g. now I did not check whether you made any change to the function other than moving it, because doing so requires too much work and I can't be bothered to invest that).

Reply:
Okay, for next time. I think we will have to do some cleanup soon anyway.